tests/settings/ConfigurationGatheringTest.py

```diff
@@ -83,13 +84,39 @@ def test_gather_configuration(self):
 
         self.assertEqual(len(local_bears['cli']), 0)
 
-    def test_default_coafile_parsing(self):
+    def test_default_coafile_deprecation(self):
+        '''
+        This test is temporary, it will send a directory
+        path which doesnot contain system_coafile named file
+        to function in Constants.py and ask for allocating
+        a system_coafile. Since there is no file nameds
+        system_coafile, it should log a warning
+        '''
```

Comment:
Optional:
I think the test is quite self-explaining mentioning some deprecation of `default_coafile`, so you don't have to put it here :)